specification/dartLangSpec.tex

```diff
@@ -1573,6 +1575,17 @@ \subsubsection{Required Formals}
 It is a compile-time error if \VAR{} occurs as
 the first token of a \synt{fieldFormalParameter}.
 
+\LMHash{}%
+It is a compile-time error if a parameter derived from
+\synt{fieldFormalParameter} occurs in a function declaration
+which is not a non-redirecting generative constructor.
```

Comment:
"in" is a little vague. Doesn't definitely rule out something like 
```dart
Foo(int foo(int this.x));
```
maybe:
> It's a compile-time error if a parameter derived from
> \sybt{fieldFormalParameter} occurs as a parameter of anything 
> except a non-redirecting generative constructor.

Reply:
Done.